src/modules/map/components/LocationPopup.vue

```diff
@@ -270,21 +270,47 @@ export default {
 <style lang="scss" scoped>
 @import 'src/scss/webmapviewer-bootstrap-theme';
 .location-popup {
+    // Triangle border
     &::before {
         $arrow-height: 15px;
         position: absolute;
         top: -($arrow-height * 2);
         left: 50%;
         margin-left: -$arrow-height;
         border: $arrow-height solid transparent;
-        border-bottom-color: $light;
+        border-bottom-color: var(--bs-border-color-translucent);
```

Comment:
as bootstrap is imported (in SCSS), you could (I think) directly use SCSS variable here.
So something like : `border-bottom-color: $border-color-translucent` (see https://getbootstrap.com/docs/5.2/utilities/borders/#css)

Reply:
Done